test/Makefile.am

```diff
@@ -12,4 +12,13 @@ wd_mempool_test_LDADD=-L../.libs -l:libwd.so.2 -l:libwd_crypto.so.2 -lnuma
 endif
 wd_mempool_test_LDFLAGS=-Wl,-rpath,'/usr/local/lib'
 
-SUBDIRS=. hisi_hpre_test hisi_sec_test hisi_zip_test
+SUBDIRS=. hisi_sec_test
+
```

Comment:
Are the test tools of the SEC module enabled by default?

Reply:
yes, sec has no requirement of libz or libcrypto(openssl)